src/components/axes/abstractAxis.ts

```diff
@@ -488,16 +488,43 @@ export module Axis {
                                     });
       var lastLabelClientRect: ClientRect;
 
-      visibleTickLabels.each(function (d: any) {
+      // Base interval value between labels
+      var interval = 1;
```

Comment:
Probably want to move this variable down to the for loop.  It's a bit out of place up here